tests/bluechi_test/service.py

```diff
@@ -81,15 +113,15 @@ def __init__(self, name: str = "simple.service") -> None:
 
 class SimpleRemainingService(SimpleService):
     """A simple service, which adds RemainAfterExit enabled to SimpleService"""
-    def __init__(self, name: str = "simple.service") -> None:
+    def __init__(self, name: str = "simple.service", directory: str = SERVICE_DIRECTORY) -> None:
         super(SimpleRemainingService, self).__init__(name)
```

Comment:
```suggestion
        super(SimpleRemainingService, self).__init__(name, directory)
```
The directory parameter needs to be passed in as well.

Reply:
Done